tensorly/decomposition/candecomp_parafac.py

```diff
@@ -7,9 +7,10 @@
 from ..kruskal_tensor import kruskal_to_tensor
 from ..tenalg import khatri_rao
 
-# Author: Jean Kossaifi <jean.kossaifi+tensors@gmail.com>
-# Author: Chris Swierczewski <csw@amazon.com>
-# Author: Sam Schneider <samjohnschneider@gmail.com>
+# Authors: Jean Kossaifi <jean.kossaifi+tensors@gmail.com>
+#          Chris Swierczewski <csw@amazon.com>
+#          Sam Schneider <samjohnschneider@gmail.com>
+#          Aaron Meurer
```

Comment:
You can add my email here too (asmeurer@gmail.com)

Reply:
Absolutely, wasn't sure you wanted your email there and didn't want to expose it in case :)